content/guides/code-splitting-libraries.md

```diff
@@ -154,9 +154,10 @@ module.exports = function() {
 ## Manifest File
 
 But, if we change application code and run `webpack` again, we see that the hash for the vendor file changes. Even though we achieved separate bundles for `vendor` and `main` bundles, we see that the `vendor` bundle changes when the application code changes.
+
 This means that we still don't reap the benefits of browser caching because the hash for vendor file changes on every build and the browser will have to reload the file.
 
-The issue here is that on every build, webpack generates some webpack runtime code, which helps webpack do its job. When there is a single bundle, the runtime code resides in it. But when multiple bundles are generated, the runtime code is extracted into the common module, here the `vendor` file.
+The issue here is that on every build, webpack generates some webpack runtime code, which helps webpack do its job. When there is a single bundle, the runtime code resides in it. But when multiple bundles are generated, the runtime code is extracted into the common module - the `vendor` file in this example.
```

Comment:
Hmmm, I see your point. Idk if our system converts the double dash to an em dash yet but it's definitely something we can tackle down the road. You could use the html entity, `&#8212;`, but that'd look rather out of place mid sentence. I would just do `--` for now as I think most people understand that's meant to be an em dash. I'll look into the actual conversion later on.